applications/conf/routes

```diff
@@ -89,6 +89,8 @@ GET        /$path<[\w\d-]*(/[\w\d-]*)?/(video|audio)/.*>/info.json
 GET        /$path<[\w\d-]*(/[\w\d-]*)?/(video|audio)/.*>.json                   controllers.MediaController.renderJson(path)
 GET        /$path<[\w\d-]*(/[\w\d-]*)?/(video|audio)/.*>                        controllers.MediaController.render(path)
 
+# Interactive service worker
+GET        /$path<[\w\d-]*(/[\w\d-]*)?(/[\w\d-]*)?>/interactive-service-worker.js           controllers.InteractiveController.renderInteractiveServiceworker(path)
```

Comment:
check that *.js isn't blocked by the vcl with only a whitelist,  it used to be, but I'm not sure now